CHANGELOG.md

```diff
@@ -2,11 +2,55 @@
 
 ## Table of Contents
 
+- [v0.3.0](#v030)
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 - [v0.1.0-rc2](#v010-rc2)
 - [v0.1.0-rc1](#v010-rc1)
 
+## v0.3.0
+
+API Version: v1alpha1
+
+### API changes
+
+#### Gateway
+- The `NoSuchGatewayClass` status reason has been deprecated.
+  [#635](https://github.com/kubernetes-sigs/gateway-api/pull/635)
+
+#### HTTPRoute
```

Comment:
@robscott should we indicate that path match is now a pointer?

xref: https://github.com/kubernetes-sigs/gateway-api/pull/564